nationalarchives/ds-wagtail
# [UN-664] Update promoted list block tracking code

Ticket URL: https://national-archives.atlassian.net/jira/software/c/projects/UN/boards/75

## About these changes

Updates the `data-component-name` attribute for the promoted list component

## How to check these changes

1. View story page locally (see http://localhost:8000/stories/alice-hawkins/#h2.read-more)
2. Inspect the 'read more' links component and see that the data-component-name attribute is showing the correct value (as per ticket requirement)

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
